_data/sites.json

```diff
@@ -5209,6 +5209,17 @@
             "sedo.com"
         ]
     },
+    
+    {
+        "name": "SpigotMC",
```

Comment:
This entry needs to be entered between the "Speaker Deck" and "Splitwise" entries to be in alphabetical order. This also causes CI to fail.